src/include/daos/checksum.h

```diff
@@ -159,12 +171,17 @@ daos_csum_type2algo(enum DAOS_CSUM_TYPE type);
  *			for it.
  * @param ft		Pointer to the function table for checksum calculations
  * @param chunk_bytes	Chunksize, typically from the container configuration
+ * @param srv_verify	whether server-side checksum verification is enabled
+ * @param dedup		whether deduplication is enabled on the server
+ * @param dedup_verify	whether to memcmp data on the server for deduplication
+ * @param dedup_bytes	deduplication size threashold in bytes
  *
  * @return		0 for success, or an error code
  */
 int
 daos_csummer_init(struct daos_csummer **obj, struct csum_ft *ft,
-		  size_t chunk_bytes, bool srv_verify);
+		  size_t chunk_bytes, bool srv_verify, bool dedup,
+		  bool dedup_verify, size_t dedup_bytes);
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?